pulp_smash/pulp2/utils.py

```diff
@@ -1,11 +1,229 @@
 # coding=utf-8
 """Utility functions for Pulp 2 tests."""
+import io
 import unittest
+from urllib.parse import urljoin, urlparse
 
 from packaging.version import Version
 
-from pulp_smash import api, config, selectors, utils
-from pulp_smash.pulp2.constants import PLUGIN_TYPES_PATH
+from pulp_smash import api, cli, config, exceptions, selectors, utils
+from pulp_smash.pulp2.constants import (
+    CONTENT_UPLOAD_PATH,
```

Comment:
Please, sort these constants alphabetically.

Reply:
These constants are already sorted alphabetically. There's nothing to do.